uint8 tag_id
string tf_id
WaypointType type